src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ToModelTransformers.kt

```diff
@@ -104,10 +106,7 @@ fun transformToLicenceSummary(
   isDueForEarlyRelease = isDueForEarlyRelease,
   isDueToBeReleasedInTheNextTwoWorkingDays = isDueToBeReleasedInTheNextTwoWorkingDays,
   updatedByFullName = licence.getUpdatedByFullName(),
-  homeDetentionCurfewActualDate = when (licence) {
-    is HdcLicence -> licence.homeDetentionCurfewActualDate
-    else -> null
-  },
+  homeDetentionCurfewActualDate = if (licence.isHdcLicence()) licence.homeDetentionCurfewActualDate else null,
```

Comment:
guess this check isn't technically required as the SentenceDateHolder would return null for none-variations. But think this is fine